keras/callbacks.py

```diff
@@ -1762,7 +1762,8 @@ class EarlyStopping(Callback):
         improvement.
     patience: Number of epochs with no improvement
         after which training will be stopped.
-    verbose: verbosity mode.
+    verbose: int. 0: quiet, 1: message when the training is stopped
+        and if weights got restored.
```

Comment:
suggest: when the weights are being restored

Reply:
My line of thought was that the message that the weights got restored is only printed if the `restore_best_weights` parameter is set to `true`. So it is not certain as "when" would imply.